FAQ.md

```diff
@@ -341,3 +347,11 @@ Note this option is not shown in the output of `etcd --help`.
 ```
 
 Indeed this distinction was eliminated by etcd in their master branch, but not backport to announced versions, so be care when deploy your etcd cluster.
+
+## How to achieve high availability of APISIX?
+
+The high availability of APISIX can be divided into two parts:
+
+>1.Data plane: The data plane of Apache APISIX is stateless and can be elastically scaled at will. Just add a layer of LB in front.
```

Comment:
As this word is not quoted from others, maybe we can use `1. Data...` directly?

Reply:
updated.